### ISSUE TITLE
add expand\collapse to tree view

### ISSUE THREAD

**Issue by RohalskyiOleksii** [testomatio/app#871]
**Is your feature request related to a problem? Please describe.**
can`t expand\collapse all test suites at once

**Describe the solution you'd like**
add expand\collapse to all places where it possible
![image](https://github.com/testomatio/app/assets/132286699/204dbec4-a51a-49df-badb-d3c94adbc078)

**Comment by TetianaKhomenko** [comment]
@RohalskyiOleksii could you please add more context, maybe some screenshots?

**Comment by RohalskyiOleksii** [comment]
hope it helps
![image](https://github.com/testomatio/app/assets/132286699/e621926e-005d-404d-9d2d-710255b078bf)

**Comment by TetianaKhomenko** [comment]
@RohalskyiOleksii I think we have such elements:

https://github.com/testomatio/app/assets/77803888/e232ba4d-2989-4daa-b77b-e05926723edf

**Comment by RohalskyiOleksii** [comment]
yes. but only one by one and not all suites at once by **1click**. 
and let's say there are 100+ suites that are collapsed after page reload and you want to see all suites tree structure, in that case expanding them one by one is very time consuming. but 2 buttons to expand\collapse all at one time by one click will solve this problem
@TetianaKhomenko
